common/configurable/src/main/java/io/helidon/common/configurable/LruCache.java

```diff
@@ -223,46 +255,4 @@ public void clear() {
     V directGet(K key) {
         return backingMap.get(key);
     }
-
-    /**
-     * Fluent API builder for {@link io.helidon.common.configurable.LruCache}.
-     *
-     * @param <K> type of keys
-     * @param <V> type of values
-     */
-    @Configured
-    public static class Builder<K, V> implements io.helidon.common.Builder<Builder<K, V>, LruCache<K, V>> {
-        private int capacity = DEFAULT_CAPACITY;
-
-        private Builder() {
-        }
-
-        @Override
-        public LruCache<K, V> build() {
-            return new LruCache<>(this);
-        }
-
-        /**
-         * Load configuration of this cache from configuration.
-         *
-         * @param config configuration
-         * @return updated builder instance
-         */
-        public Builder<K, V> config(Config config) {
-            config.get("capacity").asInt().ifPresent(this::capacity);
-            return this;
-        }
-
-        /**
-         * Configure capacity of the cache.
-         *
-         * @param capacity maximal number of records in the cache before the oldest one is removed
-         * @return updated builder instance
-         */
```

Comment:
the default value of 10K seems to have been lost in translation.

Reply:
Fixed